docs/source/cli_help/status.rst

```diff
@@ -79,3 +79,7 @@ With ``--color=always``, git machete always emits colors and with ``--color=auto
 -L, --list-commits-with-hashes    Additionally list the short hashes and messages of commits introduced on each branch.
 
 --no-detect-squash-merges         Only consider *strict* (fast-forward or 2-parent) merges, rather than rebase/squash merges, when detecting if a branch is merged into its upstream (parent).
+
+ .. note::
+
+    You can add additional space after ``└─`` and before branch name by setting git config key ``machete.status.extraSpaceBeforeBranchName``.
```

Comment:
Pls also update docs.py

Reply:
Plus also mention what might be the purpose for setting this key (easier copying on certain terminals)

Reply:
I added example, i am not 100% certain that it's necessary, @PawelLipski wdyt?
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/86779115/168641730-fdebba16-bcf7-43df-9eee-eb6abf740deb.png">

Reply:
Okay! gr8